code/core/build.gradle.kts

```diff
@@ -43,3 +43,8 @@ apiValidation {
             "com.adobe.marketing.mobile.core.BuildConfig"
     ))
 }
+
+dependencies {
+    androidTestImplementation("org.mockito:mockito-core:5.3.1")
```

Comment:
Can you use the dependency from [BuildConstants](https://github.com/adobe/aepsdk-commons/blob/dev/android/aepsdk-gradle-plugin/aepsdk-gradle-plugin/src/main/kotlin/com/adobe/marketing/mobile/gradle/BuildConstants.kt#L75) until we automatically add this from our plugin?

Reply:
It's not included in the AEP plugin's instrumentation tests. I will add a TODO that reminds us of adding it to the plugin later.